Lib/test/support/__init__.py

```diff
@@ -387,6 +387,19 @@ def wrapper(*args, **kw):
         return wrapper
     return decorator
 
+def requires_musl():
+    """Decorator raising SkipTest if the musl is not available."""
+    import subprocess
+    proc = subprocess.run(["ldd"], stderr=subprocess.PIPE, universal_newlines=True)
```

Comment:
See my comment in the conversation section. The above fails on Macs since they don't ship with `ldd `as a tool.